### ISSUE TITLE
Integrate maps view as an alternative to restaurants list

### ISSUE THREAD

**Issue by josefdolezal** [syky27/LunchGuy#16]
Hey @syky27! I would like to open discussion about map of restaurants from obedar API.

## Context

I am using LunchGuy few times a week and usually, I check only one or two restaurants, since I don't know where the rest of them is located. It would be nice to have:

1) Map view with all restaurants from API
2) Possibility to open restaurant in Google Maps/Apple Maps from menu (for details like ratings, web site, phone numbers, ...)

## Proposal

Create new `LocationProvider` protocol which will return `CLLocationCoordinate2D` for given restaurant. I saw tomaskadlec/lunch_guy#54 and tomaskadlec/lunch_guy#58 and I know that it's not currently implemented at backend. However, the list of restaurants has now less than twenty items, so we can create static provider. It would simply match restaurant by its name and return coordinates from hardcoded set. Once the backend implements this feature, we can just swap implementation of `LocationProvider`.

The UI would contain:

1) Navigation bar item in restaurant list, which would switch between map view and list view
    * The map will show pins with restaurants with possibility show its name on tap and then open menu from that
    * This is similar to Yelp app
2) Navigation bar item in restaurant menu, which would open action sheet with possibility to open Google Maps and Apple Maps

What do you think?

**Comment by syky27** [comment]
I totally agree

**Comment by syky27** [comment]
But #4 is more pressing matter right now...

**Comment by josefdolezal** [comment]
Yeah, I was also thinking about it. Since we removed most of the dependencies in last PR, I think it will be enough to just mention and link Alamofire, Fabric and Crashlytics from README. So it mostly depends on #13.

cc @hroncok

**Comment by syky27** [comment]
I believe that it needs to be mentioned inside the app to comply with those Licenses....
--
Tomas Sykora

**Comment by hroncok** [comment]
Your are slipping from the topic of this issue ;)
